src/NodeNameResolver/NodeNameResolver.php

```diff
@@ -100,11 +95,7 @@ public function isCaseSensitiveName(Node $node, string $name): bool
             return false;
         }
 
-        if ($node instanceof MethodCall) {
-            return false;
-        }
-
-        if ($node instanceof StaticCall) {
+        if ($node instanceof CallLike && !$node instanceof FuncCall) {
```

Comment:
I wonder about `New_`. was it also missing from this condition or intentionally not if-cased?

Reply:
is passing `New_` previously got the `true` or  same `false` ? that same question with `NullsafeMethodCall`

Reply:
my testing suggests it is now the same as before. the test-suite is still green, which matches my local observation.

the `New_` class also does not define a name property, so its "unnamed".

Reply:
`New_` is already instanceof `CallLike`